run_tests.sh

```diff
@@ -4,9 +4,9 @@ set -euo pipefail
 
 export DJANGO_SETTINGS_MODULE=config.settings.test
 export DJANGO_DOT_ENV_FILE=.env.test
-docker compose -f docker-compose.yml -f docker-compose.dev.yml build --force-rm db redis ganache
-docker compose -f docker-compose.yml -f docker-compose.dev.yml up --no-start db redis ganache
-docker compose -f docker-compose.yml -f docker-compose.dev.yml start db redis ganache
+docker compose -f docker-compose.yml -f docker-compose.dev.yml build --force-rm db redis ganache rabbitmq
+docker compose -f docker-compose.yml -f docker-compose.dev.yml up --no-start db redis ganache rabbitmq
+docker compose -f docker-compose.yml -f docker-compose.dev.yml start db redis ganache rabbitmq
```

Comment:
Maybe it's time to add healthcheck to docker compose now that's supported and remove the sleep: https://docs.docker.com/compose/compose-file/05-services/#healthcheck (we can do it in another PR)